aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -328,6 +328,41 @@ void add_dense_sparse_worker_cpu(Tensor& r, Scalar value, const SparseTensor& sp
   });
 }
 
+template <typename scalar_t>
+void add_dense_sparse_worker_cpu(Tensor& r, Scalar value, const SparseTensor& sparse, const Tensor& indices, const Tensor& values) {
+
+  int64_t values_dense_size = values.stride(0);
+  AT_ASSERT(values_dense_size > 0);
+  AT_ASSERT(values.is_contiguous());
```

Comment:
If `values.is_contiguous() == true` then values_dense_size is always 1, you don't need one of this line.

Reply:
@VitalyFedyunin yeah, will just keep one assert here